examples/js/effects/VREffect.js

```diff
@@ -55,34 +69,76 @@ THREE.VREffect = function ( renderer, onError ) {
 
 	// fullscreen
 
-	var isFullscreen = false;
+	var isPresenting = false;
 
 	var canvas = renderer.domElement;
 	var fullscreenchange = canvas.mozRequestFullScreen ? 'mozfullscreenchange' : 'webkitfullscreenchange';
 
-	document.addEventListener( fullscreenchange, function ( event ) {
+	document.addEventListener( fullscreenchange, function () {
+
+		if ( deprecatedAPI ) {
+
+			isPresenting = document.mozFullScreenElement || document.webkitFullscreenElement;
+
+		}
+
+	}, false );
+
+	window.addEventListener( 'vrdisplaypresentchange', function () {
 
-		isFullscreen = document.mozFullScreenElement || document.webkitFullscreenElement;
+		if (vrHMD) {
+
+			isPresenting = vrHMD.isPresenting;
+
+		}
 
 	}, false );
 
```

Comment:
I noticed that the three.js examples do not provide a method of exiting VR. so in your latest WebVR 1.0 API Chromium builds, if the developer doesn't explicitly stop presenting, the content stays frozen in the headset (until relaunching the browser - and sometimes I've noticed it takes a system restart).

how about adding this to prevent that footgun?

```js
window.addEventListener( 'beforeunload', this.exitPresent.bind(this) );
```